src/js/themes/dark.js

```diff
@@ -66,7 +66,25 @@ export default deepFreeze({
     },
   },
   icon: {
-    color: textColor,
+    extend: css`${(props) => {
+      let { color } = props.theme.icon;
+      if (props.color && props.color !== 'plain') {
+        color = colorForName(props.color, props.theme);
+      }
+      if (props.dark === colorIsDark(color)) {
+        color = props.dark ?
+          props.theme.global.colors.darkBackground.text :
+          props.theme.global.colors.lightBackground.text;
+      }
+      return `
+        fill: ${color};
+        stroke: ${color};
+      `;
+    }
+} `,
+  },
+  anchor: {
+    color: '#2b4369',
```

Comment:
This should be a formal color reference from `global.colors`.

Reply:
it was - inheriting from vanilla.js:
```
anchor: {
    textDecoration: 'none',
    fontWeight: 600,
    color: css`${props => props.theme.global.colors.brand}`,
  },
```
and we have invisible anchors on brand background:
![screenshot689](https://user-images.githubusercontent.com/6075606/39615374-c3c8c2f2-4f43-11e8-9673-5904340a480b.jpg)

https://grommet-nextjs.herokuapp.com/add-ons/numberinput?theme=dark

Reply:
Do as you please - close, reject or whatever, I am not touching this PR and not submitting anything else.
The theming in v2 based on dark/light background is a hack anyway but invisible Anchors on the brand menu seems like a major bug to me.